composer.json

```
@@ -15,7 +15,8 @@
         "symfony/property-access": "^5.4.0",
         "webmozart/assert": "^1.9",
         "phpstan/phpstan": "^1.0",
-        "friendsofphp/php-cs-fixer": "^3.10"
+        "friendsofphp/php-cs-fixer": "^3.10",
+        "symfony/console": "5.4.*"
```

Technically it is better to do that :
```suggestion
        "symfony/console": "^5.4"
```